examples/webgl2_sandbox.html

```diff
@@ -48,6 +50,8 @@
 
 			//
 
+			if ( ! Detector.webgl2 ) Detector.addGetWebGLMessage( { version: 'webgl2' });
```

Comment:
I think i would add `Detector.addGetWebGL2Message()` instead in order to avoid a parameter object.

Reply:
I agree.

Reply:
@Mugen87 @mrdoob what do you think about adding a `addGetWebGLMessageByVersion` to avoid duplicating code?
```javascript
addGetWebGLMessageByVersion: function ( parameters, version ) {

  var parent, id, element;

  parameters = parameters || {};

  parent = parameters.parent !== undefined ? parameters.parent : document.body;
  id = parameters.id !== undefined ? parameters.id : 'oldie';

  element = Detector.getWebGLErrorMessage( version );
  element.id = id;

  parent.appendChild( element );

},

addGetWebGL2Message: function ( parameters ) {
  
  Detector.addGetWebGLMessageByVersion( parameters, 'webgl2' );

},

addGetWebGLMessage: function ( parameters ) {

  Detector.addGetWebGLMessageByVersion( parameters, 'webgl' );

}
```

Do you think we should do something similar with `Detector.getWebGLErrorMessage/getWebGL2ErrorMessage` ?

Reply:
> what do you think about adding a addGetWebGLMessageByVersion to avoid duplicating code?

I think that's fine. But I would name it `_addGetWebGLMessageByVersion()` or make a function instead (and move the whole code into a closure).

> Do you think we should do something similar with Detector.getWebGLErrorMessage/getWebGL2ErrorMessage ?

 If it makes the code easier to read, why not? 😊

Reply:
+1